### PR TITLE
Allows modsuits to be set to wear on other slots.

### PR DESCRIPTION
**Pull request by JohnFulpWillard**
## About The Pull Request

Allows for MODsuit control units to be worn on slots that isn't the bag slot and actually work.
So far no MODsuits will actually use this, but I would like to add one in the future, I'm just not motivated to do the MODsuit itself, so I'm adding the support for it at least to see if it would be approved.

## Why It's Good For The Game

Lawyer tie modsuit will be real in 2023.

This doesn't really change the game much so I don't know how much I actually have to explain here.

## Changelog

Not needed.

### CODE REVIEW

File: code/modules/mod/modules/modules_supply.dm [hunk 1]
```diff
@@ -79,7 +79,7 @@
 		dropped_crate.forceMove(target_turf)
 		balloon_alert(mod.wearer, "dropped [dropped_crate]")
 		drain_power(use_power_cost)
-		mod.wearer.update_inv_back()
+		mod.wearer.update_clothing(mod.slot_flags)
```
Comment (Fikou) [inline code/modules/mod/modules/modules_supply.dm:82]: this and line 67 are things i forgot to delete can you cut them out?

**Review by Fikou — CHANGES_REQUESTED** [review]
there are multiple strings that mention the suit being back worn, you need to change those to be more generic

**Comment by Fikou** [comment]
i was intending to do this myself in a larger pr but sure i guess

File: code/modules/mod/mod_control.dm [hunk 2]
```diff
@@ -200,7 +201,7 @@
 		. += span_notice("Charge: [core ? "[get_charge_percent()]%" : "No core"].")
 		. += span_notice("Selected module: [selected_module || "None"].")
 	if(!open && !active)
-		. += span_notice("You could put it on your <b>back</b> to turn it on.")
+		. += span_notice("You could equip it to turn it on.")
```
Comment (Fikou) [inline code/modules/mod/mod_control.dm:204]: can you make this only run if there's no wearer?

**Review by Fikou — APPROVED** [review]

**Comment by Palpetinus** [comment]
Would this also make it possible for modsuits to affect other slots but only head, suit, gloves and shoes? Or is that still the strict "requirement" of what makes a modsuit a modsuit?

**Comment by Fikou** [comment]
> Would this also make it possible for modsuits to affect other slots but only head, suit, gloves and shoes? Or is that still the strict "requirement" of what makes a modsuit a modsuit?

that requires a large refactor that will be coming in some months probably

File: code/modules/mod/modules/modules_supply.dm [hunk 3]
```diff
@@ -79,7 +78,6 @@
 		dropped_crate.forceMove(target_turf)
 		balloon_alert(mod.wearer, "dropped [dropped_crate]")
 		drain_power(use_power_cost)
-		mod.wearer.update_inv_back()
```
Comment (LemonInTheDark) [inline code/modules/mod/modules/modules_supply.dm:82]: Were these redundant?

Comment (JohnFulpWillard) [reply code/modules/mod/modules/modules_supply.dm:82]: Fikou told me in one of their reviews to remove it, so I suppose so.

**Pull request merged**